src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -37,14 +41,74 @@ export class QueryRunner extends Component<IQueryRunnerProps, IQueryRunnerState>
     }
   };
 
+  private handleOnEditorChange = (body?: string) => {
+    if (body) {
+      this.setState({ sampleBody: body });
+    }
+  };
+
+  private handleOnHeaderNameChange = (name?: any) => {
+    if (name) {
+      this.setState({
+        headerName: name,
+      });
+    }
+  };
+
+  private handleOnHeaderValueChange = (value?: any) => {
+    if (value) {
+      this.setState({
+        headerValue: value,
+      });
+    }
+  };
+
+  private handleOnHeaderDelete = (headerIndex: any) => {
+    const { headers } = this.state;
+    const headersToDelete = [...headers];
```

Comment:
Add a new line after variable declarations.

Here we are destructuring headers then inititializing `headersToDelete` with `headers`. The same can be achieved succinctly by doing the following

```
  const { headers: headersToDelete } = this.state

```
This will destructure and inititialize